### PR TITLE
Gpc remove

### PR DESCRIPTION
**Pull request by emulgeator**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		"lukasoppermann/http-status": "^2.0.1"
 	},
 	"require-dev": {
-		"mikey179/vfsStream": ">=1.1.0",
+		"mikey179/vfsstream": ">=1.1.0",
```
Comment (adam-rocska) [inline composer.json]: Dafuq 🤣

File: src/YapepBase/Request/HttpRequest.php [hunk 2]
```diff
@@ -97,21 +97,8 @@ class HttpRequest implements IRequest {
 	 * @param array $server               The $_SERVER array
 	 * @param array $env                  The $_ENV array
 	 * @param array $files                The $_FILES array
-	 * @param bool  $magicQuotesEnabled   If TRUE, the get, post, cookie arrays will be recursively stripped of slashes.
 	 */
-	public function __construct(
```
Comment (adam-rocska) [inline src/YapepBase/Request/HttpRequest.php:102]: ❤️

File: .gitignore [hunk 3]
```diff
@@ -6,3 +6,5 @@
 vendor
 composer.phar
 .DS_Store
+.php_cs.cache
+.phpunit.result.cache
```
Comment (adam-rocska) [inline .gitignore]: Wouldn't it be a good idea to disable phpunit result caching all-in-all? We used to have a lot of issues with it in our legacy project in the past. (differet fw, different setup)

File: composer.json (hunk 1 above)
Comment (szeber) [reply composer.json]: This has been done in 0.3 already... not that it matters

**Review by szeber — APPROVED** [review]
This looks fine, but shouldn't it be targetting 0.3 instead?

File: composer.json (hunk 1 above)
Comment (emulgeator) [reply composer.json]: yeah that target should have been 0.3.x

File: .gitignore (hunk 3 above)
Comment (emulgeator) [reply .gitignore]: In the new version we could turn it off. Doesn't matter that much actually

**Pull request merged**